src/main.rs

```diff
@@ -153,20 +153,17 @@ fn run_app(terminal: &mut DefaultTerminal, mut app: App, tick_rate: Duration) ->
                 Mode::FileTree => {
                     if !file_tree.loaded() {
                         while let Ok(e) = f_rx.try_recv() {
-                            match e {
-                                Some(file) => {
-                                    file_tree.add_file(file);
-                                }
-                                None => {
-                                    file_tree = file_tree.clone().finish();
-                                    break;
-                                }
+                            if let Some(file) = e {
+                                file_tree.add_file(file);
+                            } else {
+                                file_tree = file_tree.clone().finish();
+                                break;
```

Comment:
This is just a stylistic choice. I usually don't care which is used, but here I find the match visually easier to grasp. I know `clippy::pedantic` gives the change as a suggestion